src/matchers/utils.ts

```diff
@@ -113,4 +113,26 @@ export const getElementText = async (
   throw new Error(`Invalid input length: ${args.length}`)
 }
 
-export const quote = (val: string | null) => `'${val}'`
+export const quote = (val: string | null) =>
+  val === null ? "null" : `'${val}'`
+
+export const getMessage = (
+  { isNot, promise, utils }: jest.MatcherContext,
```

Comment:
We also gave the user ability to use `expect-playwright` without `jest`. And don't we get any errors in this case?

Reply:
Interesting, I didn't realize until now that was an option (glossed over it in the readme).  What are the use cases for using it without Jest?

Reply:
We allow it to use it in https://github.com/microsoft/playwright-test since its based on https://github.com/microsoft/folio. So we support the use-case of using it fully standalone or with the http://npmjs.com/package/expect library which is also used in folio.

Reply:
Sorry for not responding sooner, got sidetracked with other priorities at work.  My plan is to do some additional testing with folio later this week.

Reply:
idk but i think that we will need the check if `jest` is available, and there is no `jest` - just return regular message, just how we do earlier

Reply:
Okay, so I had a chance to test this out.  Turns out the utils I'm using in this PR are available when using `expect.extend` but not the custom expect wrapper that this library is exporting.  If we could update to use a strategy more like this code:

```ts
import { it, expect } from "@playwright/test";
import matchers from "expect-playwright/lib/matchers";

expect.extend(matchers);

it("is a basic test with the page", async ({ page }) => {
  await page.goto("https://playwright.dev/");
  await expect(page).toEqualText("h3", "Playwrights");
});
```

Then it would actually work.  Here is what that looks like:

<img width="944" alt="Screen Shot 2021-05-20 at 3 49 44 PM" src="https://user-images.githubusercontent.com/25914066/119047311-5c942880-b983-11eb-8e3b-6fd08f88bc05.png">

The TS definitions would need updating which looks similar to jest:

```ts
declare namespace folio {
  interface Matchers<R> extends PlaywrightMatchers<R> {}
}
```

Reply:
Looks good, regrading types, in the new folio version which gets released soon it will automatically pick up our existing Jest extend types.